Dockerfile

```diff
@@ -1,60 +1,24 @@
+ 
 FROM gcr.io/deeplearning-platform-release/tf2-gpu.2-2
 WORKDIR /root
 
 # Installs sndfile library for reading and writing audio files 
 RUN apt-get update && apt-get install -y libsndfile-dev
 
-# Installs Magenta DDSP library and upgrades Tensorflow
+# Upgrades Tensorflow and Tensorflow Probability
 # Newer version of Tensorflow is needed for multiple VMs training
-RUN pip install --upgrade pip && pip install --upgrade tensorflow ddsp
+RUN pip install --upgrade pip && pip install --upgrade tensorflow tensorflow-probability
 RUN pip show tensorflow
 
-# Installs Google Cloud SDK, this is mostly for using gsutil to copy the configs.
-RUN wget -nv \
-    https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz && \
-    mkdir /root/tools && \
-    tar xvzf google-cloud-sdk.tar.gz -C /root/tools && \
-    rm google-cloud-sdk.tar.gz && \
-    /root/tools/google-cloud-sdk/install.sh --usage-reporting=false \
-        --path-update=false --bash-completion=false \
-        --disable-installation-options && \
-    rm -rf /root/.config/* && \
-    ln -s /root/.config /config && \
-    # Remove the backup directory that gcloud creates
-    rm -rf /root/tools/google-cloud-sdk/.install/.backup
+# Install cloudml-hypertune package needed for hyperparameter tuning
+RUN pip install cloudml-hypertune
 
-# Path configuration
-ENV PATH $PATH:/root/tools/google-cloud-sdk/bin
-# Makes sure gsutil will use the default service account
-RUN echo '[GoogleCompute]\nservice_account = default' > /etc/boto.cfg
+RUN wget https://github.com/ana-simionescu/ddsp/archive/master.zip
+RUN unzip master.zip
+RUN cd ddsp-master && python setup.py install
```

Comment:
To avoid checking in temporary files in the fs, merge these 3 commands in a single one:
```
RUN wget https://github.com/ana-simionescu/ddsp/archive/master.zip && \
  unzip master.zip && \
  cd ddsp-master && python setup.py install